spec/predicates/simple-verification.md

```diff
@@ -0,0 +1,109 @@
+# Predicate type: Simple Verification
+
+Type URI: https://in-toto.io/attestation/simple-verification/v0.1
+
+Version: 0.1
+
+Authors: Tom Hennen (@TomHennen), Andrew McNamara (@arewm)
+
+## Purpose
+
+A Simple Verification Attestation (SVA) communicates that an artifact has been evaluated against one or more policies, and records the properties that were verified, at the point-in-time of the evaluation. This enables consumers to trust a concise summary of verification results, without requiring access to all underlying attestations or policies.
+
+## Use Cases
+
+-   Allowing software consumers to delegate complex policy decisions to a trusted verifier, and rely on a simple summary of which properties were verified.
+-   Supporting evolving verification needs (e.g., SLSA, vulnerability scanning, organizational policies) without coupling the SVA format to any specific framework.
+-   Enabling verifiers to add additional information (e.g., policy references, expiration, property-level timing) as needed, while maintaining a simple core schema.
+
+## Prerequisites
+
+This predicate depends on the [in-toto Attestation Framework]. Familiarity with the concept of attestation statements and the use of URIs for identifying verifiers and resources is assumed.
+
+## Model
+
+An SVA is an attestation that a verifier (`verifier.id`) has checked an artifact (`resourceUri`) at a specific time (`timeVerified`) and asserts a set of verified properties (`properties`). The SVA does not include information for reproducing the verification result. Instead, verifiers MAY generate additional provenance attestations for the simple verification attestation as needed.
+
+## Schema
+
+```jsonc
+{
+  // Standard attestation fields:
+  "_type": "https://in-toto.io/Statement/v1",
+  "subject": [{ ... }],
+
+  // Predicate:
+  "predicateType": "https://in-toto.io/attestation/simple-verification/v0.1",
+  "predicate": {
+    "verifier": {
+      "id": "<URI>"
+    },
+    "timeVerified": "<TIMESTAMP>",
```

Comment:
Just a nit, but could "timeCreated" be a better name?

Since the checks that produce the properties are not pre-determined, different properties can be produced based on quite diverse processes. For example, some properties may be "inherited" from verification that happened a long time ago; some others may not be a result of a single verification; etc.

Hence "verified" could be mis-interpreted by consumers.

I think this attribute should only designate the time that a particular SVA is created.

Reply:
I like that clarity. Thanks for the suggestion.